src/shared/browser/__test__/chrome.test.ts

```diff
@@ -0,0 +1,139 @@
+import { afterEach } from 'node:test'
+import { describe, expect, test, vi } from 'vitest'
+
+import { chromeBrowser } from '../chrome'
+
+describe('browser', () => {
+  describe('storage', () => {
+    type StorageValue = object
+
+    const KEY = 'testKey'
+    const storedValue: StorageValue = { someData: 'value' }
+    const defaultValue: StorageValue = { defaultData: 'defaultValue' }
+    const circularValue = { prop: 'value', circularRef: {} }
+    circularValue.circularRef = circularValue
+    const invalidJSONString = `{"name": "Joe", "age": null]`
+
+    describe('local', () => {
+      describe('get', () => {
+        test('should get default value if storage is empty', async () => {
+          const data = await chromeBrowser.storage.local.get(KEY, defaultValue)
+
+          expect(data).toEqual(defaultValue)
+        })
+
+        test('should get data', async () => {
+          await chromeBrowser.storage.local.set<StorageValue>(KEY, storedValue)
+
+          const data = await chromeBrowser.storage.local.get(KEY, defaultValue)
+
+          expect(data).toEqual(storedValue)
+        })
+      })
+
+      describe('set', () => {
+        test('should set data to the storage', async () => {
```

Comment:
i think we can leave this test and delete the above one because they are the same